core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -466,6 +466,23 @@ class StandaloneDynamicAllocationSuite
     }
   }
 
+  test("kill all executors on localhost") {
```

Comment:
I'll refactor these tests when I start adding more tests for configuration parameters and the like.